boards/itsybitsy_m0/Cargo.toml

```diff
@@ -42,7 +46,7 @@ embedded-graphics = "0.6.0"
 
 [features]
 # ask the HAL to enable atsamd21g18a support
-default = ["rt", "atsamd-hal/samd21g18a"]
+default = ["rt", "atsamd-hal/samd21g18a", "atsamd-hal/unproven"]
```

Comment:
im fine with this and did it for pygamer some time ago, but we should check with some other users to make sure it doesnt bother anyone. Unproven goes away entirely very shortly anyway either way.